### PR TITLE
ZK-4798: setMold("paging") causes exception while checkbox is selected

### PR DESCRIPTION
**Pull request by plijyfes**

### CODE REVIEW

File: zul/src/archive/web/js/zul/mesh/MeshWidget.js [hunk 1]
```diff
@@ -603,36 +603,43 @@ zul.mesh.MeshWidget = zk.$extends(zul.Widget, {
...
 	},
 	/** Returns the number of pages.
 	 * Note: there is at least one page even no item at all.
 	 * @return int
 	 * @see Paging#getPageCount
 	 */
 	getPageCount: function () {
-		return (this.paging || this._paginal).getPageCount();
+		var pgnl = this.paging || this._paginal;
+		return pgnl ? pgnl.getPageCount() : 0;
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/mesh/MeshWidget.js:625]: 1

File: zul/src/archive/web/js/zul/mesh/MeshWidget.js [hunk 2]
```diff
@@ -603,36 +603,43 @@ zul.mesh.MeshWidget = zk.$extends(zul.Widget, {
...
 	},
 	/** Returns the active page (starting from 0).
 	 * @return int
 	 * @see Paging#getActivePage
 	 */
 	getActivePage: function () {
-		return (this.paging || this._paginal).getActivePage();
+		var pgnl = this.paging || this._paginal;
+		return pgnl ? pgnl.getActivePage() : -1;
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/mesh/MeshWidget.js:633]: 0

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by scribetw — APPROVED** [review]

**Review by DevChu — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**